src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -107,14 +107,26 @@
 						</template>
 						{{ t('spreed', 'Join open conversations') }}
 					</NcActionButton>
+
+					<NcActionButton v-if="canModerateSipDialOut"
```

Comment:
btw this is enough for Desktop client 18 talking to Talk 17 ? Or do we need more capability checking?

Reply:
getCapabilities()?.spreed?.config.call['sip-dialout-enabled'] should be enough?

Reply:
if enough question marks are used, I guess yes